e2e/nextjs/project.json

```diff
@@ -2,12 +2,13 @@
   "root": "e2e/nextjs",
   "sourceRoot": "e2e/nextjs/src",
   "projectType": "library",
+  "implicitDependencies": ["@griffel/webpack-loader"],
```

Comment:
As we don't `import`/`require` `@griffel/webpack-loader` anywhere - Nx fails to detect it as a dependency.